### PR TITLE
m_k112 is m_disableShakeToggled

### PR DESCRIPTION
**Pull request by MuhXd**

### CODE REVIEW

File: bindings/2.206/GeometryDash.bro [hunk 1]
```diff
@@ -7881,7 +7881,7 @@ class GJGameLevel : cocos2d::CCNode {
 	int m_workingTime2;
 	bool m_lowDetailMode;
 	bool m_lowDetailModeToggled;
-	bool m_k112;
+	bool m_disableShaketoggled;
```
Comment (matcool) [inline bindings/2.206/GeometryDash.bro:7884]: m_disableShake**T**oggled

Comment (MuhXd) [reply bindings/2.206/GeometryDash.bro:7884]: what? it is

Comment (MuhXd) [reply bindings/2.206/GeometryDash.bro:7884]: i checked i needed it

Comment (matcool) [reply bindings/2.206/GeometryDash.bro:7884]: the T should be uppercase

Comment (MuhXd) [reply bindings/2.206/GeometryDash.bro]: i'm blind lol

**Pull request merged**